ivadomed/loader/loader.py

```diff
@@ -661,7 +706,13 @@ def __getitem__(self, index):
         Args:
             index (int): Slice index.
         """
-        seg_pair_slice, roi_pair_slice = self.indexes[index]
+
+        # If self.length in model_params, extract pair slices subimages from handlers
+        if self.length:
```

Comment:
Related to the question regarding the "default values" for no patching.
Also, here we are testing length, but what happen in no striding in model_params? to be discussed?

Reply:
I think we should support both (no param or length_2d=[] as Charley suggested). So maybe add a condition and len(self.length)

Reply:
I think `if self.length` will return false for both `None` and `[]`. However, I use the condition a few times, maybe I should put it in a bool variable ex: `is_2d_patch`, at the beginning.

For testing the dimensions and values of length and stride, I was planning on doing it in `prepare_indices` (similar to the 3D version). If I use a bool variable, maybe I should test it at the same time.

Reply:
I like this:
```
put it in a bool variable ex: is_2d_patch, at the beginning
```